src/inputs/dropdown/search-dropdown/story-helper.ts

```diff
@@ -197,6 +198,15 @@ const extraArgTypes: ArgTypes = {
             type: { summary: null },
         },
     },
+    'selected-extra': {
```

Comment:
nitpick: This blocks duplication of prop name and slot name. What about changing it to 'selectedExtraSlot'?